scripts/evan.access.js

```diff
@@ -0,0 +1,253 @@
+
+    config.accounts.push(c.id)
+    config.accountMap[c.id] = c.privateKey
+    config.keyConfig[sha3(c.id)] = c.profileKey
+    config.keyConfig[sha9(c.id, c.id)] = c.profileKey
+  }
+
+  Object.assign(accessProfiles, accountConfig)
+  Object.assign(createProfiles, createConfig)
+
+
+  for(const a in accessProfiles)
+    mapAccount(a, accessProfiles[a])
+
+  // first fetch the accounts to create from the original runtime config
+  for(let mn in config.aliases) {
+    const alias = config.aliases[mn]
+    const cc = createdProfiles.accounts[alias]
+    if(cc) {
+      mapAccount(a,cc)
+      console.log('Already created, loading ', alias)
+    }
+  }
+
+  let create = {}
+  for(const a in createProfiles) {
+    const c = createProfiles[a]
+    const cc = createdProfiles[a]
+
+    if(cc) mapAccount(a,cc)
+    if(!c.mnemonic || !c.password || !c.alias) console.log('skipped creating account ', a)
+    else {
+      config.ensureProfiles = true
+      // set up profile creation fields in config
+      create[a] = c
+
+      config.mnemonics[c.mnemonic] = c.password
+      config.aliases[c.mnemonic] = c.alias
+      if(c.contacts && c.contacts.length) {
+        const key = c.id || c.mnemonic
+        config.contactConfig[key] = []
+        for(let contactName of c.contacts) {
+          const contact = accessProfiles[contactName] || createProfiles[contactName]
+          config.contactConfig[key].push(contact.id || contact.mnemonic)
+        }
+      }
+    }
+  }
+
+  config.activeAccount = config.activeAccount || config.accounts[0]
+
+  return config
+}
+
+// creating profiles, is done by profiles-helper
+// in this file we only load profiles and parse/edit configs
+
+async function cacheProfiles(config) {
+
+  const createRTCache = {}
+  for(let mn in config.mnemonic2account) {
+    const a = config.mnemonic2account[mn]
+
+    let addToCache = true
+
+    for(let cp in createdProfiles) if(cp.id === a) { addToCache = false; break }
+
+    if (addToCache) {
+
+      console.log('caching ', config.aliases[mn],'/',a)
+      createRTCache[config.aliases[mn]] = {
+        id: a,
+        alias: config.aliases[mn],
+        mnemonic: mn,
+        password: config.mnemonics[mn],
+        privateKey: config.accountMap[a],
+        profileKey: config.keyConfig[sha3(a)],
+        contacts: [],
+      }
+    }
+  }
+
+  if(Object.keys(createRTCache).length) {
+    Object.assign(createdProfiles, createRTCache)
+
+    return pfy(fs.writeFile)( __dirname + '/config/createdProfiles.json', JSON.stringify(createdProfiles,null,2))
+  }
+}
+
+
+// this just loads existing accounts, cached or preconfigured
+// if you want to create profiles, don't use this, it never loads the profile-helpers
+
+async function init(cfg = {}) {
+  // if we really want to support mulitple different blockchain cores with different cfg at the same time,
+  // we need a real stack to manage this
+  ++busy
```

Comment:
```suggestion
  busy += 1
```
see: http://jslint.com/help.html#inc and https://eslint.org/docs/rules/no-plusplus